README.md

```diff
@@ -26,14 +18,35 @@ Conjur provides:
 
 * try Conjur: [Start Here](https://try.conjur.org)
 * [support](https://try.conjur.org/support.html)
-* [API Documentation][api-doc]
+* [API Documentation][api]
+
+[![Join Conjur Slack](https://img.shields.io/badge/slack-open-e01563.svg)][slack]
+[![Conjur API Ruby Gem](https://img.shields.io/gem/dt/conjur-api.svg?label=ruby%20gem%20downloads)][gem]
```

Comment:
Let's not mix conjur and other tools here. I think if we properly funnel to the docsite, and the urls are like `/integrations/conjur-api-ruby` that should be fine. Too many shields clutters up this README is all. This shield should go in the README in `cyberark/conjur-api-ruby`.

Reply:
Okay, I'll create PRs in those repos for their respective badges